### ISSUE TITLE
WAV2VEC: Unsupervised Pre-Training for Speech Recognition

### ISSUE THREAD

**Issue by jaehwlee** [jaehwlee/paper-summary#1]
[link](https://arxiv.org/abs/2006.11477)
Summary
* pre-training을 통해서 mel-spec 대신 speech data 자체의 general representation을 학습하자.

Approach 
* Speech계의 BERT를 만들어보자, 단 방식은 word2vec과 유사.

Model
![image](https://user-images.githubusercontent.com/33409264/103438650-4279d800-4c78-11eb-91d2-a71b71869914.png)
* raw waveform에서 어떤 샘플이 올지 예측은 데이터 분포를 알아야 하기 때문에 힘듦.
* 저차원의 latent space에서 density ratio를 모델링하는 방식으로 접근.
* X는 raw waveform, Z는 latent space, C는 context space
* 학습 종료 후 이 C로 전처리를 하는 것, 1.6kHz 기준 30ms가 하나의 벡터로 표현됨.

Loss Function
![image](https://user-images.githubusercontent.com/33409264/103438673-87057380-4c78-11eb-9bd4-61301ada77e4.png)
* 틸다 z는 negative sample, z_i는 현 위치 샘플, z_i+k는 k시점 이후의 샘플, c_i는 현 시점 컨텍스트 벡터
* 현 시점의 컨텍스트 벡터와 z_i+k가 맞는 쌍인지 아닌지 가려내서 positive쌍이면 두 벡터 간의 거리가 줄어드는 방식으로 학습 진행
* negative라는 뜻은 컨텍스트 벡터에 맞지 않는 샘플이라는 뜻, word2vec에 해당 개념 있음, 이 논문에선 distractor라는 용어로 사용

Result
![image](https://user-images.githubusercontent.com/33409264/103438791-51fa2080-4c7a-11eb-8ba4-f4ad206eb435.png)
* baseline은 80 log-mel filterbank coefficient로 전처리한 waveform
* AM은 wav2letter++, LM은 표에 나온 4가지 디코딩 모델 사용
* pretrained 모델을 썼을 때, 학습에 쓰인 데이터가 많을수록 성능이 좋음

Conclusion
* 역시 페이스북.. 정말 좋은 논문이라고 느껴짐. wav2cec 2.0도 꼭 읽어봐야겠다.
* pre-train이 아니라 data-driven한 방식과 성능 차이는 얼마나 날까?
